src/webgl/texture.js

```diff
@@ -112,6 +112,13 @@ export const TEXTURE_FORMATS = {
   // [GL.COMPRESSED_RGBA_ATC_INTERPOLATED_ALPHA_WEBGL]: {compressed: true, gl1: ETC1}
 };
 
+const V3_TO_V4_OPTIONS_MAP = {
```

Comment:
Either way is fine, not a big difference but I would opt for GL ENUM style, couple of reasons
1. Don't see much advantage other than shorter names.
2. Since we are using enums in other places, lets keep same style every where.